onebuild/commands/init_command.py

```diff
@@ -0,0 +1,8 @@
+from onebuild.commands.command import Command
+from onebuild.file_writer import write_default_config_file
+
+
+class InitCommand(Command):
+
+    def execute(self, arg_parser, arguments, build_file_name, command_name):
+        write_default_config_file(arguments[1])
```

Comment:
Cool. Makes sense. I would still like to keep filewriter separate to do the actual I/O. This execute method can have all the logic and pass the finished result to file writer which will be dumb. 

Please let me know what you think about this. I am pushing the changes.

Reply:
Ok. then let's keep this in utils instead of new file for better testing.

Reply:
Can you check the current implementation once? I feel having a file_writer would be good because in the next story ( set / unset feature ) we would need to do multiple things like append to a file or modify a file. So the interface for all of this becomes file_writer.

Reply:
Make sense. Let's keep `file_writer` as of now, and let's see what we can do after `set-unset`